src/transformStateWithClones.js

```diff
@@ -6,8 +6,31 @@
  *
  * @return {Object[]}
  */
+
 function transformStateWithClones(state, actions) {
-  // write code here
+  let ResultList = [];
```

Comment:
should be in camelCase
```suggestion
  let resultList = [];
```